app.yml

```diff
@@ -39,13 +39,14 @@ default_events:
 - pull_request_review
 - pull_request_review_comment
 - push
-# - release
+- release
```

Comment:
This is not part of the contributors.md above, is it missing?

Reply:
Ops that was me testing, it should be commented.